src/test/java/com/github/susom/database/test/CommonTest.java

```diff
@@ -1503,19 +1563,60 @@ public Date process(Rows rs) throws Exception {
     assertEquals(new Long(1L), db.toSelect("select count(*) from dbtest where d=?").argDate(dbNow).queryLongOrNull());
   }
 
+  @Test
+  public void dbDate() {
+    new Schema()
+            .addTable("dbtest")
+            .addColumn("d").asDbDate().table().schema()
+            .execute(db);
+
+    LocalDate dateOfBirth = LocalDate.of(1951, Month.AUGUST, 9);
+    db.toInsert("insert into dbtest (d) values (?)")
+            .argDbDate(dateOfBirth)
+            .insert(1);
+
+    LocalDate testDate = db.toSelect("select d from dbtest").queryDbDateOrNull();
+    assertEquals(dateOfBirth, testDate);
+  }
+
+
+  @Test
+  public void dbDateRoundTrip() {
```

Comment:
The thing I really worry about with these, which I haven't figured out how to test well, is combinations of JVM and database time zones, and edge cases like daylight savings shifts. Especially when you are trying to store a calendar day in a database column that is actually a point in time (e.g. Oracle DATE), things go spectacularly wrong in rare circumstances.

I have seen a couple strategies to help mitigate those, each of which has its own problems: a) use a column the database will definitely not treat as a datetime (e.g. integer for Julian days, or string like yyyymmdd), or b) mandate the database be in GMT/UTC and set time component to noon rather than truncating to midnight, so DST or TZ or leap second shifts don't move you to a different calendar day.